### PR TITLE
LPS-0 DO NOT MERGE Run all forms tests

### PR DESCRIPTION
**Pull request by alinedoleron**

### CODE REVIEW

**Comment by alinedoleron** [comment]
ci:test

**Comment by clovisdasilvaneto** [comment]
Hey @alinedoleron Can you rebase this code and send this PR again?

**Comment by alinedoleron** [comment]
ci:test

**Comment by alinedoleron** [comment]
ci:test

File: modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/form_builder_field_settings_sidebar.js [hunk 1]
```diff
@@ -241,6 +241,8 @@ AUI.add(
 					_changeFieldTypeMenu: function(fieldType) {
 						var instance = this;
 
+						instance._disableHeader();
```
Comment (clovisdasilvaneto) [inline modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/form_builder_field_settings_sidebar.js]: Hey @alinedoleron what do you think we rename _disableHeader for _disableFieldTypeMenu or _disableSidebarHeader or _disableSidebarMenu, as result of the menus from the sidebar will be disabled, right?

File: modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/form_builder_field_settings_sidebar.js [hunk 2]
```diff
@@ -382,17 +381,14 @@ AUI.add(
 										var previousFieldName = previousSettingsFormFieldContext.fieldName;
 
 										if (!(fieldName === 'type') && fieldName === previousFieldName) {
```
Comment (clovisdasilvaneto) [inline modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/form_builder_field_settings_sidebar.js]: Hey @alinedoleron could you group that expressions: `fieldName === previousFieldName` with parentheses?

**Pull request closed without merging**